bin/ember-template-lint.js

```diff
@@ -7,14 +7,27 @@ var path = require('path');
 var glob = require('glob');
 var Linter = require('../lib/index');
 var linter = new Linter();
+const chalk = require('chalk');
 
 function printErrors(errors) {
   if (process.argv.indexOf('--json') + 1) {
     console.log(JSON.stringify(errors, null, 2));
   } else {
+    let count = 0;
+
     Object.keys(errors).forEach(filePath => {
-      console.log(Linter.errorsToMessages(errors[filePath]));
+      let options = {};
+      let fileErrors = errors[filePath] || [];
+      count += fileErrors.length;
+
+      if (process.argv.includes('--verbose')) {
```

Comment:
Per http://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes, Node 4 does not have a `Array.prototype.includes` method.